modules/acm_sku/src/ProductOptionsManagerInterface.php

```diff
@@ -31,4 +31,34 @@ public function loadProductOptionByOptionId($attribute_code, $option_id, $langco
    */
   public function synchronizeProductOptions();
 
+  /**
+   * Create product option if not available or update the name.
+   *
+   * @param string $langcode
+   *   Lang code.
+   * @param int $option_id
+   *   Option id.
+   * @param string $option_value
+   *   Value (term name).
+   * @param int $attribute_id
+   *   Attribute id.
+   * @param string $attribute_code
+   *   Attribute code.
+   * @param int $weight
+   *   Taxonomy term weight == attribute option sort order.
+   *
+   * @return \Drupal\taxonomy\Entity\Term|null
+   *   Term object or null.
+   */
+  public function createProductOption($langcode, $option_id, $option_value, $attribute_id, $attribute_code, $weight);
```

Comment:
theoretically, it's fine but this looks BC break to me.

Reply:
added a wrapper